dace/transformation/dataflow/strip_mining.py

```diff
@@ -137,9 +137,11 @@ class StripMining(transformation.Transformation):
     new_dim_prefix = Property(dtype=str,
                               default="tile",
                               desc="Prefix for new dimension name")
-    tile_size = Property(dtype=str,
-                         default="64",
-                         desc="Tile size of strip-mined dimension")
+    tile_size_or_number = Property(
```

Comment:
The name now is hard to remember when using the API. Maybe we should keep it tile_size

Reply:
It will also create some compatibility issues with current users of dace, which use "tile_size" as a property already.

Reply:
Makes sense, I on the fence about changing it. Should it still be used as the number of tiles? Otherwise we would need a new Property again.